bitnami/kafka/Chart.yaml

```diff
@@ -42,4 +42,4 @@ maintainers:
 name: kafka
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kafka
-version: 25.1.12
+version: 25.2.2
```

Comment:
This should be `25.3.0`

Reply:
https://github.com/bitnami/charts/blob/716e86ef9c8191f9fbb598e4d7a3b0317cb48fea/bitnami/kafka/Chart.yaml#L45

Reply:
Nevermind, I could resolve the conflict during a rebase 🚀